web/package.json

```diff
@@ -94,12 +94,14 @@
     "mobile-detect": "^1.4.4",
     "next": "9.5.5",
     "next-i18next": "^7.0.1",
+    "next-pino": "^0.0.1",
```

Comment:
I think this dependency is unmaintained. Last commit was in 2018, and I got this warning:
```
warning " > next-pino@0.0.1" has incorrect peer dependency "pino@^4.13.0".
```

Seems to be working for now, but we might need to find a replacement.